### PR TITLE
Feature/modal open update

### LINKED ISSUE #7: Background scrolls instead of modal
When the modal is open, scrolling seems to affect the background only - the background will scroll if it's overflowing, and the modal won't scroll at all, whether it overflows or not. 

This can be fixed by manually using jQuery to add the 'modal-open' class to the body tag like Bootstrap does when a modal is opened, but it would be nice to see this built in to the addDialog function.

### PR DESCRIPTION
**Pull request by kevcjones-archived**
This should resolve #7 - tested in my own project and indeed it does lock the background scrolling while the modal is open.

### CODE REVIEW

**Comment by celsomtrindade** [comment]
Did you changed/commited any src files? I'm not seeing any changes on src files related to this functionality.

**Comment by kevcjones-archived** [comment]
Yeah this is when you shouldn't burn the midnight oil and push work up. Changes actually there now. My only concern right now, is that the demo package  is a bit flimsy, you have to run it with the older cli... I've tested this change in my own angular@4.3.2 project and it works as expected.

File: src/dialog-holder.component.ts [hunk 1]
```diff
@@ -79,12 +82,27 @@ export class DialogHolderComponent {
...
   }
 
+  /**
+   * Bind a body class 'modal-open' to a condition of dialogs in pool > 0
+   */
+  
+  private updateBodyClass() {
+    const bodyClass = 'modal-open';
+    const body = document.getElementsByTagName('body')[0];
+    if(this.dialogs.length == 0) {
```
Comment (celsomtrindade) [inline src/dialog-holder.component.ts]: I would use `===` to check this. Personal preference. Or even `if(!this.dialogs.length)`.

But, nice work. I'll test it on my project, it's using version 5.

**Review by celsomtrindade — APPROVED** [review]
So far so good! Tried on my Angular 5 project and it seems to be working as expected.

**Pull request merged**